on ln. 29, +0.0
	of __init__(...)
	| bound successfully to localhost:50808
on ln. 41, +56.06
	of handle_connections(...)
	| received connection from 127.0.0.1
on ln. 137, +58.07
	of handle_http_requests(...)
	| [127.0.0.1:34096] received 'GET /../ht'
on ln. 50, +72.33
	of handle_connections(...)
	| user-interrupt caught
	| fatal error, exiting ...
on ln. 59, +72.34
	of __del__(...)
	| garbage collecting SocketServer instance
